File: Java/src/SqrtXTest.java
import org.junit.Assert;
import org.junit.Test;

public class SqrtXTest {

    @Test
    public void testMySqrt() {
        SqrtX sx = new SqrtX();

        Assert.assertTrue(sx.mySqrt(0) == 0);

        Assert.assertTrue(sx.mySqrt(1) == 1);

        Assert.assertTrue(sx.mySqrt(2147395599) == 46339);

        Assert.assertTrue(sx.mySqrt(2147395600) == 46340);

        Assert.assertTrue(sx.mySqrt(6) == 2);
    }
    
}

File: Kotlin/src/SqrtXTest.kt
import org.junit.Test

class SqrtXTest {

    @Test
    fun testMySqrt() {
        val sx = SqrtX()
        
        assert(sx.mySqrt(0) == 0)

        assert(sx.mySqrt(1) == 1)

        assert(sx.mySqrt(2147395599) == 46339)

        assert(sx.mySqrt(2147395600) == 46340)

        assert(sx.mySqrt(6) == 2)
    }

}